### ISSUE TITLE
VoiceOver doesn't report elements from outside of focused <webview>

### ISSUE THREAD

**Issue by juliekoubova** [electron/electron#25274]
### Preflight Checklist
* [x] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/master/CONTRIBUTING.md) for this project.
* [x] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [x] I have searched the issue tracker for an issue that matches the one I want to file, without success.

### Issue Details
* **Electron Version:** 10.1.1
* **Operating System:** macOS 10.15.6
* **Last Known Working Electron version:** N/A, it also reproduces in 8.5.0

### Expected Behavior
VoiceOver should always report the same list of landmarks, regardless of whether the focus is inside a `<webview>` or not.

### Actual Behavior
When the focus is inside of a `<webview>`, VoiceOver only reports landmarks from that `<webview>` and not from the outer window.

### To Reproduce
1. Load the following Fiddle: https://gist.github.com/c8bcc2290d883724a55ee3063b55cc93
1. Press `Command`+`F5` to enable VoiceOver
1. Focus the form controls that are outside the `<webview>`.
    - Press `Ctrl`+`Option`+`U` to show the list of landmarks, press right arrow until you see the list of headings.
    - Verify that you see all headings, including the "*Example Domain*" from the `<webview>`.
1. Focus inside the `<webview>` (e.g. on the "*More Information link*")
    - Press `Ctrl`+`Option`+`U` to show the list of landmarks, press right arrow until you see the list of headings.
    - You should see the exactly same list of headings as before, but this time, all the headings from the main window are missing.

**Comment by felixrieseberg** [comment]
Thanks for the detailed report (and the also super detailed other report about a11y and `<webview>`. I appreciate the work you put into these reports, so I wanted to let you know that we're currently advising developers to _not_ use the `<webview>`. It's a brittle piece of technology and you'll almost certainly be better off with iframes or frames.

The companies sponsoring the majority of the work on Electron currently don't use `<webviews>` for that exact reason, so it'll likely be a long time, if ever, before subtle issues with it are fixed.

**Comment by juliekoubova** [comment]
@felixrieseberg I know 😢  Sadly it's not really an option for us to stop using `<webview>` right now.

**Comment by juliekoubova** [comment]
This [also reproduces in Chrome](https://gist.github.com/juliekoubova/f21b440150d470818b8905b821c71d16) using regular `<iframe>`. 

➡️   &nbsp;[Chromium Issue 1124703: Elements preceding focused IFrame are missing from VoiceOver landmark list](https://bugs.chromium.org/p/chromium/issues/detail?id=1124703)
